dev-python/patch-ng/patch-ng-1.17.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+inherit distutils-r1
+
+DESCRIPTION="Library to parse and apply unified diffs, fork of dev-python/patch"
+HOMEPAGE="https://github.com/conan-io/python-patch-ng https://pypi.org/project/patch-ng/"
+SRC_URI="https://github.com/conan-io/python-patch-ng/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="test? ( dev-python/coverage )"
+
+S="${WORKDIR}/python-${P}"
+
+python_test() {
+	coverage run tests/run_tests.py || die "Tests failed"
```

Comment:
Coverage testing is usually meaningless for us, since it checks code quality and doesnt run python runtime testing. Looking at https://github.com/conan-io/python-patch-ng/blob/master/tests/run_tests.py it seems to use unittest instead.